.github/workflows/integration-test.yml

```diff
@@ -77,6 +77,7 @@ jobs:
           - test: 'logstash'
           - test: 'loki'
           - test: 'mongo'
+          - test: 'nats'
```

Comment:
I think the non-docker-compose integration tests need additional setup on the VM. I believe one of @jdrouet  goals in converting them was to leave this extra setup out to speed it up.

I think we could re-add the separate block for non-docker-compose integration tests, with the setup, or convert the nats integration tests to the docker-compose style.